modules/images-configuration-registry-mirror.adoc

```diff
@@ -0,0 +1,182 @@
+to pull images from a repository on a source image registry and have
+it resolved by a repository on a mirrored image registry.
+* Identify multiple mirrored repositories for each target
+repository, to make sure that if one mirror is down, another
+can be used.
+
+Here are some of the attributes of repository mirroring in {product-title}:
+
+* Image pulls are resilient to registry downtimes
+* Disconnected clusters can request to pull
+images from critical locations (such as quay.io)
+and have registries behind a company firewall
+provide the requested images.
+* A particular order of registries is tried when
+an image pull request is made, with the permanent
+registry typically being the last one tried.
+
+Setting up repository mirroring can be done in the following ways:
+
+* At {product-title} installation time: By pulling container images needed
+by {product-title} and then bringing those images behind your company's
+firewall, you can install {product-title} into a datacenter that is
+disconnected from the Internet. See
+link:https://github.com/openshift/openshift-docs/blob/636ca5e931e8e3000349649d86f280a814de25c4/modules/installation-mirror-repository.adoc[Mirroring the {product-title} image repository]
+for details.
+
+* After {product-title} installation time: Even if you don't configure mirroring during {product-title} 
+installation, you can do so later using the ImageContentSourcePolicy object.
+
+The following procedure provides a post-installation mirror configuration, where you create an ImageContentSourcePolicy object that identifies:
+
+* The source of the container image repository you want to mirror
+* A separate entry for each mirror repository you want to offer the content
+requested from the source repository.
+
+The mirror information you enter is added to the `/etc/containers/registries.conf`
+file on every node in the {product-title} cluster.
+
+When a node makes a request for an image from the source repository, it
+tries each mirrored repository in turn until it finds the requested on content.
+If they all fail, the cluster tries the source repository. Upon success, the
+image is pulled to the node.
+
+.Prerequisites
+* Have administrative privilege to a working {product-title} cluster.
+
+.Procedure
+
+. Configure mirrored repositories. To do that, you can either:
++
+* Set up a mirrored repository with Red Hat Quay, as
+described in link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/manage_red_hat_quay/repo-mirroring-in-red-hat-quay[Red Hat Quay Repository Mirroring].
+Using Red Hat Quay allows you to copy images from one repository to another,
+but also automatically sync those repositories repeatedly over time.
+* Use a tool such as `skopeo` to copy images manually
+from the source directory to the mirrored repository.
++
+For example, after installing the skopeo RPM package
+on a RHEL 7 or RHEL 8 system, use the `skopeo` command as shown in this example:
++
+----
+$ skopeo copy \
```

Comment:
Are skopeo and podman available by default in both RHEL 7 and 8? If not, I'd make getting them a prereq.

Reply:
They are  available in both RHEL 7 and RHEL 8. And there is no trick to installing them, since each is in an RPM package of its own name. But it does raise a question of whether or not we should add a prerequisite for installing commands that are an optional way to do a task. I'll leave it out for now.